### PR TITLE
updated venues.json sde locations

### LINKED ISSUE #3783: SDE3 venues wildly incorrect
The data for a bunch of SDE venues, including:

- [ ] https://nusmods.com/venues/SDE3-BARM
- [ ] https://nusmods.com/venues/SDE3-LT421
- [ ] https://nusmods.com/venues/SDE3-LT422
- [ ] https://nusmods.com/venues/SDE3-LT423
- [ ] https://nusmods.com/venues/SDE3-LT424 (this could possibly be correct, but worth checking again)

look very wildly wrong. I personally know little about SDE but some of these LTs are placed in Tembusu College / Central Library which feels unlikely to me.

It'd be great if someone could track down and update the data for these venues!

### PR DESCRIPTION
**Pull request by yk-tuturu**
## Context
 closes #3783

## Implementation
 updated x and y coordinates for SDE3-LT421, LT422, LT423, and SDE-BARM in venues.json

### CODE REVIEW

File: website/package.json [hunk 1]
```diff
@@ -173,5 +173,6 @@
   },
   "browserslist": [
     "extends browserslist-config-nusmods"
-  ]
+  ],
+  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
```
Comment (zwliew) [inline website/package.json:177]: Please remove this change

**Review by zwliew — CHANGES_REQUESTED** [review]

**Comment by zwliew** [comment]
Hey, thanks for helping out. Just a minor issue with the change -- otherwise, it looks good to me!

**Comment by yk-tuturu** [comment]
Hi, I've fixed the problem woooops

**Review by zwliew — APPROVED** [review]
Thanks again!

**Pull request merged**